# KorstanjeLab_Publications
This a public repository containing all the tools and resources used in publications out of the Korstanje Lab. 
Please do not push anything unpublished for the time being. 
If you need help please reach out to one of the team maintainers [Ron Korstanje](https://github.com/rkorstan), [Susan Sheehan](https://github.com/susan-sheehan), or [Yuka Takemon](https://github.com/ytakemon).

## Publications
(coming soon to a repo near you)

## Submodule References
[DOQTL](https://github.com/dmgatti/DOQTL)  
Gatti, D. M., Svenson, K. L., Shabalin, A., Wu, L.-Y., Valdar, W., Simecek, P., et al. (2014). Quantitative trait locus mapping methods for diversity outbred mice. G3 (Bethesda, Md.), 4(9), 1623–1633. http://doi.org/10.1534/g3.114.013748 

[GBRS](https://github.com/churchill-lab/gbrs) **(Primary methods manuscript currently in preperation, but used in: )**  
Chick, J. M., Munger, S. C., Simecek, P., Huttlin, E. L., Choi, K., Gatti, D. M., et al. (2016). Defining the consequences of genetic variation on a proteome-wide scale. Nature, 534(7608), 500–505. http://doi.org/10.1038/nature18270

[EMASE](https://github.com/churchill-lab/emase) **(Primary methods manuscript currently in preperation, but used in: )**  
Munger, S. C., Raghupathy, N., Choi, K., Simons, A. K., Gatti, D. M., Hinerfeld, D. A., et al. (2014). RNA-Seq alignment to individualized genomes improves transcript abundance estimates in multiparent populations. Genetics, 198(1), 59–73. http://doi.org/10.1534/genetics.114.165886

